java/mvn/examples/spring_boot/cloudbuild.yaml

```diff
@@ -0,0 +1,6 @@
+steps:
+- name: 'gcr.io/cloud-builders/java/mvn'
```

Comment:
my testing shows that this build takes ~14 seconds, and if I substitute it with `name: maven` and `entrypoint: mvn`, it takes 25 seconds. So, good!